tools/gen-random-bout/gen-random-bout.cpp

```diff
@@ -42,13 +42,25 @@ static void push_obj(KTest *b, const char *name, unsigned non_zero_bytes,
   o->bytes = (unsigned char *)malloc(o->numBytes);
 
   unsigned i;
-  for(i = 0; i < non_zero_bytes; i++)
-    o->bytes[i] = random();
+  for (i = 0; i < non_zero_bytes; i++) {
+    long int r = random();
+    o->bytes[i] = r ? r : 1;
```

Comment:
Good catch here!  But a better way to do this is `o->bytes[i] = random() % 255 + 1`